util/jsonutil/jsonutil.go

```diff
@@ -0,0 +1,57 @@
+package jsonutil
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+)
+
+var comma = []byte(",")[0]
+
+func DropElement(extension []byte, elementName string) ([]byte, error) {
+	buf := bytes.NewBuffer(extension)
+	dec := json.NewDecoder(buf)
+	var startIndex int64
+	var i interface{}
+	for {
+		token, err := dec.Token()
+		if err == io.EOF {
```

Comment:
Do we want to return any signal that the element was not found?

Reply:
We don't have a need for a signal currently. We can add in the feature now and maybe use it in the future, or I also support deferring this feature until/if we need it.